mail/alpine/Portfile

```diff
@@ -16,7 +16,7 @@ long_description    Alpine is a tool for reading, sending, and \
                     managing electronic messages that was designed \
                     with novice users in mind.
 
-master_sites        http://alpine.x10host.com/alpine/release/src
+master_sites        http://alpineapp.email/alpine/release/src
```

Comment:
this should ideally use `https`

Reply:
Well, that one is a bit embarrassing.  I totally didn't notice that.